django_project/frontend/src/containers/MainPage/SideBar/LayerFilterTabs.tsx

```diff
@@ -9,6 +9,22 @@ import Filter from './Filter';
 
 function LayerFilterTabs(props: { selectedMainTabIdx: number }) {
     const [selectedTabSideBar, setSelectedTabSideBar] = useState(0);
+    const [containsCharts, setContainsCharts] = useState(false);
```

Comment:
I think it's better to use something like 'showLayerFilter' instead, because it's more scalable.